spec/tictactoe_clojure/game_spec.clj

```diff
@@ -0,0 +1,16 @@
+(ns tictactoe-clojure.game-spec
+  (:require [speclj.core :refer :all]
+            [tictactoe-clojure.console :as console]
+            [tictactoe-clojure.game :refer :all]))
+
+(describe "tictactoe-clojure.game"
+  (with new-board [1 2 3 4 5 6 7 8 9])
+  (with first-player #tictactoe_clojure.player.Human{:piece "X"} )
+  (with second-player #tictactoe_clojure.player.Human{:piece "O"})
+  (with players [@first-player @second-player])
+
+  (describe "#play-game"
+    (it "loops through a game where the human enters 1, 2, 4 and loses to the computer"
+      (with-redefs [console/display (fn [_] "Player O wins")]
+      	(with-out-str (with-in-str "1\n4\n2\n5\n3\n"
+          (should= "Player O wins" (play-game @players @new-board))))))))
```

Comment:
No, initially my earlier test which was done with TDD was tied closer to the implementation of the code. That was in the iteration where I was using `should-invoke` to assert if the play-game method was calling my `take-two-move` function. I understood that testing the code that way was brittle. This was my attempt to refactor the test to test for behaviors.  But from your comment on my mvp branch, I see how this was dependent on the computer-player making specific moves. I will refactor these test.